### PR TITLE
add setting of MyProxy CaSubjectDN

### PR DESCRIPTION
**Pull request by gtallan**
This adds the option to set [MyProxy].CaSubjectDN for the case where MyProxy is not running on the local node.

### CODE REVIEW

File: manifests/config.pp [hunk 1]
```diff
@@ -90,6 +90,7 @@
     globus_connect_config { 'MyProxy/ServerBehindNAT': value => $globus::myproxy_server_behind_nat }
     globus_connect_config { 'MyProxy/CADirectory': value => $globus::myproxy_ca_directory }
     globus_connect_config { 'MyProxy/ConfigFile': value => $globus::myproxy_config_file }
+    globus_connect_config { 'MyProxy/CaSubjectDN': value => $globus::myproxy_ca_subject_dn }
```
Comment (treydock) [inline manifests/config.pp]: Would need to wrap this in if statement so only manage this resource if `$globus::myproxy_ca_subject_dn` is not `undef`

**Comment by treydock** [comment]
Is this an undocumented config option?  I don't see it in my configs.

File: manifests/config.pp (hunk 1 above)
Comment (gtallan) [reply manifests/config.pp]: Good point, I should have tested more thoroughly!

**Comment by gtallan** [comment]
> Is this an undocumented config option? I don't see it in my configs.

It's not in the default annotated globus-connect-server.conf but I found documentation for it here:

https://docs.globus.org/globus-connect-server-installation-guide/

in the section on adding additional i/o nodes (part 3b). They do sort of acknowledge it being missing from the default config file by writing "If this option isn’t present in your globus-connect-server.conf file then you will need to add it."

**Comment by treydock** [comment]
Released as 2.1.0

**Pull request merged**